Library/OcConfigurationLib/OcConfigurationLib.c

```diff
@@ -453,22 +455,29 @@ mMiscConfigurationSecuritySchema[] = {
   OC_SCHEMA_STRING_IN  ("Vault",                OC_GLOBAL_CONFIG, Misc.Security.Vault),
 };
 
+STATIC
+OC_SCHEMA
+mMiscConfigurationSerialCustomSchema[] = {
+  OC_SCHEMA_INTEGER_IN ("BaudRate",               OC_GLOBAL_CONFIG, Misc.Serial.Custom.BaudRate),
```

Comment:
Shall we replace all these with _OPT?

Reply:
No, only the dict itself. If the dict is specified, one must pass all.